requirements.txt

```diff
@@ -7,6 +7,7 @@ coloredlogs==15.0
 cryptography==3.3.2
 docutils==0.16
 Faker==6.1.1
+furo==2020.10.15b13
```

Comment:
This should be in requirements_dev.txt, right? We don't need the theme to run cci, but only if you're building the docs